.erb-lint.yml

```diff
@@ -0,0 +1,40 @@
+---
+EnableDefaultLinters: true
+linters:
+  DeprecatedClasses:
+    enabled: true
+    addendum: "Reference: https://sarken.github.io/docs/front_end_coding/classes.html."
+    rule_set:
+      - deprecated: ["bookmarks", "collections", "readings", "works"]
+        suggestion: "Avoid the plural form of these classes."
+  Rubocop:
+    enabled: true
+    rubocop_config:
+      inherit_from:
+        - .rubocop.yml
+      Layout/ArgumentAlignment:
+        EnforcedStyle: with_fixed_indentation
```

Comment:
Which style do we like more for view files? `with_fixed_indentation` is like this:

https://github.com/otwcode/otwarchive/blob/057b4ff672ef4ef480b9b2fdddb710e1b052cba7/app/views/abuse_reports/new.html.erb#L65-L66

while the default (`with_first_argument`) is like this:

https://github.com/otwcode/otwarchive/blob/057b4ff672ef4ef480b9b2fdddb710e1b052cba7/app/views/abuse_reports/new.html.erb#L58-L59

Reply:
Let's go with fixed, just because it gives you more room on any given line.